tests/appsec/test_event_tracking.py

```diff
@@ -148,6 +150,7 @@ def test_custom_event_event(self):
 
         def validate_custom_event_tags(span):
             expected_tags = {
+                "_dd.appsec.events.system_tests_event.sdk": "true",
```

Comment:
since this was never defined, we can consider it optional and remove it ? if so all other languages will pass